### PR TITLE
qm-shutdown: add page

### PR DESCRIPTION
**Pull request by byjujohn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by byjujohn** [comment]
Relates to #8537

File: pages/linux/qa-shutdown.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# qm shutdown
+
+> Shutdown a virtual machine on QEMU/KVM Virtual Machine Manager.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Shutdown a virtual machine:
+
+`qm shutdown {{VM_ID}}`
+
+- Shutdown a virtual machine after wait for at most 10 seconds:
```
Comment (kbdharun) [inline pages/linux/qa-shutdown.md:10]:
```suggestion
- Shutdown a virtual machine after waiting for at most 10 seconds:
```
I think this would be better.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. I have a small suggestion for this page.

File: pages/linux/qa-shutdown.md (hunk 1 above)
Comment (byjujohn) [reply pages/linux/qa-shutdown.md:10]: Thank you @kbdharun

**Review by einverne — APPROVED** [review]
Looks good to me, thanks @byjujohn

**Pull request merged**